### DISCUSSION TITLE
图片合成失败

### DISCUSSION THREAD

**Discussion by LINCHOR** [cinit/QAuxiliary#860] [Q&A] [answered]
开启后qq自带相机可以拍照但点击发送时显示图片合成失败

qa版本1.4.9.r1949.d308dcf
qq版本V 9.0.20.15515

qa重置插件所有功能后问题一样存在，开启安全模式可以正常发送

k50跟8t都这样

**Comment by HdShare** [comment, marked as answer]
下载重定向 功能导致的
